estate/views/estate_menu_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem
+            id="estate_menu_root"
+            name="Estate">
+    </menuitem>
+    <!-- Properties menu -->
+    <menuitem
+            id="estate_first_level_menu"
+            name="Properties"
+            parent="estate_menu_root"/>
```

Comment:
You can do submenus like this: https://github.com/odoo/odoo/blob/dff0a35413c0bcb106d0ab1086465f9200e25c5d/addons/sale/views/sale_menus.xml#L14
instead of using parent, it is more in line with modern odoo